src/peep/staff.c

```diff
@@ -372,6 +372,35 @@ void game_command_fire_staff_member(int *eax, int *ebx, int *ecx, int *edx, int
 	*ebx = 0;
 }
 
+void game_command_move_staff_member(int *eax, int *ebx, int *ecx, int *edx, int *esi, int *edi, int *ebp)
+{
+	if (*ebx & GAME_COMMAND_FLAG_APPLY) {
+		int x = *eax;
+		int y = *ecx;
+		int z = *edi;
+
+		uint16 spriteIndex;
+		rct_peep *peep = NULL;
+		FOR_ALL_STAFF(spriteIndex, peep)
```

Comment:
Yeah, the staff_ids don't get messed up even if desync'd because the hiring and firing is already done through game commands. If they use the sprite index, its very common that it will either crash the other side or move something other than the staff (once during testing I was moving around a bit of puke instead)